### PR TITLE
Enable support for new cops introduced by rubocop v0.90

### PR DESCRIPTION
**Pull request by ivgiuliani**
- `Lint/DuplicateRequire`
 - `Lint/EmptyFile`
 - `Lint/TrailingCommaInAttributeDeclaration`
 - `Lint/UselessMethodDefinition`
 - `Style/CombinableLoops`
 - `Style/KeywordParametersOrder`
 - `Style/RedundantSelfAssignment`
 - `Style/SoleNestedConditional` (disabled by default)

### CODE REVIEW

File: rubocop.yml [hunk 1]
```diff
@@ -290,3 +302,15 @@ Style/OptionalBooleanParameter:
...
+  Enabled: true
+
+Style/KeywordParametersOrder:
+  Enabled: true
+
+Style/RedundantSelfAssignment:
+  Enabled: true
+
+Style/SoleNestedConditional:
+  Enabled: false
```
Comment (ivgiuliani) [inline rubocop.yml:316]: Disabled this one... It's useful in theory, but in practice it tries to make code like this a one liner even though it's much easier to understand this way:

```ruby
if admin?
  return default_error unless user_in_staff_permissions_group?(value)
end
```

File: rubocop.yml [hunk 2]
```diff
@@ -290,3 +302,15 @@ Style/OptionalBooleanParameter:
 
 Style/StringConcatenation:
   Enabled: false
+
+Style/CombinableLoops:
+  Enabled: true
```
Comment (matt-thomson) [inline rubocop.yml:307]: Feels like this might have some unintended consequences, but happy to turn it on and find out...

**Review by matt-thomson — APPROVED** [review]
👍

File: rubocop.yml (hunk 2 above)
Comment (ivgiuliani) [reply rubocop.yml:307]: I tested this branch against PS, this cop only breaks once so we should be fine

**Pull request merged**